client/components/Reviews/ratingBreakdown.jsx

```diff
@@ -0,0 +1,66 @@
+/* eslint-disable no-restricted-syntax */
+import React from 'react';
+import metaData from './sampleRatingsData';
+import ProgressBar from './progressBar';
+import StarRating from './StarRating';
+
+function Ratings() {
+  // calculating percent of recommendations
+  const yes = Number(metaData.recommended.true);
+  const no = Number(metaData.recommended.false);
+  const total = yes + no;
+  const percent = Math.round((yes / total) * 100);
+
+  // calculating average rating for product
+  const scores = metaData.ratings;
+  const sum = 1 * (Number(scores['1'])) + 2 * (Number(scores['2'])) + 3 * (Number(scores['3'])) + 4 * (Number(scores['4'])) + 5 * (Number(scores['5']));
+  const weightedAvg = (sum / total);
+  const avgRating = Number((weightedAvg).toFixed(1));
+
+  const indiRatings = Object.values(scores);
+  console.log(indiRatings);
+  // calculate and create new array for average rating for each star display
+  const starPercentArray = [];
+  for (const score in scores) {
+    const starValue = Number(scores[score]);
+    const starPercent = Math.round((starValue / total) * 100);
+    starPercentArray.push(starPercent);
+  }
+
+  return (
+    <div style={{ fontSize: '15px', fontWeight: '1em', display: 'inline' }}>
```

Comment:
noticed that you broke out your styles in another file. might be easier to manage if they were all broken out since you mentioned you were having a little trouble with styling. suggestion only, no need to update it if you don't think it'll help!

Reply:
Hey! yes that's something I was wondering. I'll keep that it mind and take a call again on it, thanks!